src/settings.php

```diff
@@ -17,7 +17,7 @@
         ],
 
         'db' => [
-            'host' => 'mysql:host=db;',
+            'host' => 'mysql:host=127.0.0.1;',
```

Comment:
Please change the DB creds back to the original db

Reply:
done